content/300-guides/200-deployment/201-serverless/400-deploy-to-aws-lambda.mdx

```diff
@@ -1,301 +1,199 @@
 ---
-title: 'Deploy to AWS Lambda'
-metaTitle: 'Deploy to AWS Lambda'
-metaDescription: 'Learn how to deploy Node.js and TypeScript applications based on Prisma Client to AWS Lambda.'
+title: 'Deploy to AWS Lambda with Serverless Framework'
+metaTitle: 'Deploy your Serverless Framework project with Prisma'
+metaDescription: 'Learn how to avoid potential pitfalls when deploying your Serverless Framework project using Prisma'
 ---
 
 <TopBlock>
 
-This guide explains how to set up and deploy a serverless Node.js REST API to [AWS Lambda](https://aws.amazon.com/lambda/) with the [Serverless Framework](https://www.serverless.com/framework/).
-
-AWS Lambda is part of the AWS cloud platform. You can use it to run your code in a serverless environment. To deploy a REST API to AWS Lambda, you must also use [S3](https://aws.amazon.com/s3/) to host the files and [API Gateway](https://aws.amazon.com/api-gateway/) to expose the API over HTTP.
```

Comment:
Might consider keeping this sentence (or reuse in AWS Lambda page)

Reply:
I think this info is good to know and important, but doesn't fit this guide. This info would be good for the overall AWS lambda page.